### DISCUSSION TITLE
Store data on systems without filesystem

### DISCUSSION THREAD

**Discussion by WolleKette** [micropython/micropython#11415] [General]
Hi at all,

I have a NUCLEO_L432KC board, which does not support a filesystem because of few flash and I use a Micropython build with my own frozen modules.

Now I need to store some configuration data in my program and I don't know how to do that without a file system.
Please, can someone help me?

Unfortunately I can't get the MP version of the [mindforger/micropython-L432KC](https://github.com/mindforger/micropython-L432KC) project compiled. It's supposed to provide a minimal file system.

**Comment by WolleKette** [comment]
Has finished

**Comment by jimmo** [comment]
> I have a NUCLEO_L432KC board, which does not support a filesystem because of few flash and I use a Micropython build with my own frozen modules.

@WolleKette The default configuration for this board does include a 26kiB filesystem.

The only other solution is to add external storage, either external spiflash (which you can put a filesystem on) or and i2c eeprom (which you can just read/write bytes to).
